NitroxServer/GameLogic/Entities/Spawning/BatchEntitySpawner.cs

```diff
@@ -359,8 +359,51 @@ private List<Entity> ConvertComponentPrefabsToEntities(List<PrefabAsset> prefabs
                              counter++,
                              parent);
 
-                prefabEntity.ChildEntities = ConvertComponentPrefabsToEntities(prefab.Children, prefabEntity, deterministicBatchGenerator);
+                if (prefab.Name.Contains("(Placeholder)"))
+                {
+                    PrefabAsset spawnablePrefab = spawnablePrefabs.Find((x) => x.TransformAsset == transform);
+                    if (spawnablePrefab != null)
+                    {
+                        Optional<UweWorldEntity> opWorldEntity = worldEntityFactory.From(spawnablePrefab.ClassId);
 
+                        if (!opWorldEntity.HasValue)
+                        {
+                            Log.Debug("Unexpected Empty WorldEntity! " + spawnablePrefab.ClassId);
```

Comment:
String concat is bad, we should always use string interpolation

Reply:
lol  yea that part is actually copy pasted from further up in the code so I will fix both.

Reply:
532514d